detekt.yml

```diff
@@ -0,0 +1,381 @@
+autoCorrect: true
+failFast: true
+
+#test-pattern: # Configure exclusions for test sources
+#  active: true
+#  patterns: # Test file regexes
+#    - '.*/test/.*'
+#    - '.*Test.kt'
+#    - '.*Spec.kt'
+#  exclude-rule-sets:
+#    - 'comments'
+#  exclude-rules:
+#    - 'NamingRules'
+#    - 'WildcardImport'
+#    - 'MagicNumber'
+#    - 'MaxLineLength'
+#    - 'LateinitUsage'
+#    - 'StringLiteralDuplication'
+#    - 'SpreadOperator'
+#    - 'TooManyFunctions'
+
+build:
+#  warningThreshold: 5
+#  failThreshold: 10
```

Comment:
✏️ I think `warningThreshold` and `failThreshold` are deprecated. I think we should remove it instead of commenting it out so we don't forget and use it.